common/djangoapps/microsite_configuration/tests/test_backends.py

```diff
@@ -0,0 +1,428 @@
+# -*- coding: utf-8 -*-
+"""
+Test Microsite backends.
+"""
+import logging
+from mock import patch
+
+from django.conf import settings
+from django.test import TestCase
+
+from microsite_configuration.backends.base import AbstractBaseMicrositeBackend, BaseMicrositeTemplateBackend
+from microsite_configuration.backends.filebased import SettingsFileMicrositeBackend
+from microsite_configuration import microsite
+from microsite_configuration.models import (
+    Microsite,
+    MicrositeTemplate,
+    MicrositeHistory,
+)
+from microsite_configuration.tests.tests import (
+    DatabaseMicrositeTest,
+)
+from microsite_configuration.tests.factories import (
+    MicrositeFactory,
+    MicrositeTemplateFactory,
+)
+
+log = logging.getLogger(__name__)
+
+
+class NullBackend(AbstractBaseMicrositeBackend):
+    """
+    A class that does nothing but inherit from the base class
```

Comment:
Can you add a little information here about why it's important for this class to exist?